assets/css/homepage.css

```diff
@@ -0,0 +1,530 @@
+  }
+
+  .headline-header h1 {
+    font-weight: 700;
+    font-size: 35px;
+    line-height: 40px;
+  }
+
+  .headline-desc {
+    margin-bottom: 10px;
+    padding-left: 20px;
+  }
+
+  .headline-desc p {
+    font-size: 14px;
+    line-height: 20px;
+  }
+
+  .headline-info h5 {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 20px;
+    letter-spacing: 0;
+  }
+
+  .headline-info p {
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 20px;
+  }
+
+  .headline-desc,
+  .headline-info {
+    width: 100%;
+  }
+}
+
+#program {
+  background: url('../img/program-bg.jpg') no-repeat right;
+  background-size: cover;
+  width: 100%;
+  padding: 40px 140px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.program-header {
+  margin-bottom: 30px;
+}
+
+.program-header h2,
+.speakers-header h2,
+.partners-header h2 {
+  color: #fff;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 30px;
+  letter-spacing: 0;
+}
+
+.program-header h2::after,
+.speakers-header h2::after,
+.partners-header h2::after {
+  content: '';
+  display: block;
+  position: absolute;
+  width: 50px;
+  border-bottom: 3px solid #ec5242;
+  margin-left: 45px;
+  margin-top: 10px;
+}
+
+.program-info {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.program-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 20px 12px;
+  height: 285px;
+  margin-bottom: 20px;
+  background-color: rgb(213, 216, 222);
+  background-color: rgba(213, 216, 222, 0.2);
+  margin-right: 4px;
+}
+
+.program-card:hover {
+  border: 2px solid #fff;
+}
+
+.program-image {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.program-image img {
+  height: 50px;
+}
+
+.program-title {
+  width: 100%;
+}
+
+.program-title h4 {
+  color: #ec5242;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 30px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.program-desc p {
+  color: #fff;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 20px;
+  text-align: center;
+}
+
+.program-image,
+.program-desc,
+.program-title {
+  margin-bottom: 10px;
+}
+
+.program-link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.program-link a {
+  color: #fff;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 22px;
+  letter-spacing: 0;
+}
+
+.program-link-1 {
+  text-decoration: underline;
+}
+
+.program-link-2 {
+  display: none;
+  background: #ec5242;
+  justify-content: center;
+  align-items: center;
+  padding: 15px;
+  margin-top: 7px;
+}
+
+@media screen and (max-width: 768px) {
+  #program {
+    padding: 10px 20px;
+    justify-content: flex-start;
+  }
```

Comment:
- [ ] As well, here, for the mobile design, please give it a bit more padding top and bottom to the whole section and a bit more padding top to the button, in order to match the Figma design;

Your implementation | Figma Design
:------:|:------:
![image](https://user-images.githubusercontent.com/83596135/169172761-b3e3e58f-d6a3-4620-9a74-b979c5ae6acb.png) | ![image](https://user-images.githubusercontent.com/83596135/169172814-0bce98b1-1db8-4b08-9487-bdd5ebfea51a.png)